tests/forms_tests/tests/test_formsets.py

```diff
@@ -1180,49 +1107,46 @@ def test_html_safe(self):
         self.assertEqual(str(formset), formset.__html__())
 
 
-data = {
-    'choices-TOTAL_FORMS': '1',  # the number of forms rendered
-    'choices-INITIAL_FORMS': '0',  # the number of forms with initial data
-    'choices-MIN_NUM_FORMS': '0',  # min number of forms
-    'choices-MAX_NUM_FORMS': '0',  # max number of forms
-    'choices-0-choice': 'Calexico',
-    'choices-0-votes': '100',
-}
-
+class FormsetAsTagTests(SimpleTestCase):
+    def setUp(self):
+        self.data = {
+            'choices-TOTAL_FORMS': '1',
+            'choices-INITIAL_FORMS': '0',
+            'choices-MIN_NUM_FORMS': '0',
+            'choices-MAX_NUM_FORMS': '0',
+            'choices-0-choice': 'Calexico',
+            'choices-0-votes': '100',
+        }
+        self.management_form_html = (
+            '<input type="hidden" name="choices-TOTAL_FORMS" value="1">'
+            '<input type="hidden" name="choices-INITIAL_FORMS" value="0">'
+            '<input type="hidden" name="choices-MIN_NUM_FORMS" value="0">'
+            '<input type="hidden" name="choices-MAX_NUM_FORMS" value="0">'
+        )
 
-class FormsetAsFooTests(SimpleTestCase):
     def test_as_table(self):
-        formset = ChoiceFormSet(data, auto_id=False, prefix='choices')
+        formset = ChoiceFormSet(self.data, auto_id=False, prefix='choices')
         self.assertHTMLEqual(
             formset.as_table(),
-            """<input type="hidden" name="choices-TOTAL_FORMS" value="1">
-<input type="hidden" name="choices-INITIAL_FORMS" value="0">
-<input type="hidden" name="choices-MIN_NUM_FORMS" value="0">
-<input type="hidden" name="choices-MAX_NUM_FORMS" value="0">
+            self.management_form_html + """
```

Comment:
I'm not sure whether it should be considered part of a separate clean up across all cases of this in the code base or whether we can handle this here for all of the adjacent lines, but it would be nice to fix the indentation:
```python
            self.management_form_html + (
                '<tr><th>Choice:</th><td><input type="text" name="choices-0-choice" value="Calexico"></td></tr>'
                '<tr><th>Votes:</th><td><input type="number" name="choices-0-votes" value="100"></td></tr>'
            )
```

Reply:
@pope1ni Indentation is wrong in many places in this file. I don't think it is worth to fix it, but I also agree that we can fix indentation in places where we already changing tests :+1: